test/check-machines-consoles

```diff
@@ -744,6 +745,102 @@ fullscreen=0
         b.wait_visible(".vm-console-vnc canvas")
         self.assertIn("password=on", m.execute(f"grep ^-vnc /var/log/libvirt/qemu/{name}.log"))
 
+        # Change global password, but without restarting the VM. This
+        # means that Cockpit will use the new password when
+        # connecting, but the VM still expects the old.
+        m.execute("sed -i -e 's/vnc_password = \"barfoo\"/vnc_password = \"foobar\"/' /etc/libvirt/qemu.conf")
```

Comment:
:warning: destructive!

`self.sed_file`?

Reply:
My thinking was that this modifies a file that has been installed via `self.write_file` earlier, and will already be restored. Is that not safe?